### DISCUSSION TITLE
stellarium  on windows 11

### DISCUSSION THREAD

**Discussion by eriktim82** [Stellarium/stellarium#3697] [General]
I've in countering a issue where stellarium fails to load and just shows a white screen.  

if its a fresh install everything it fine, but if i save settings then close out and reopen is when i encounter this issue.   

happens every time and is repeatable.  I've even tried didn't versions of stellarium

**Comment by gzotti** [comment]
And your CPU/GPU/Stellarium version/logfile are ... ? https://github.com/Stellarium/stellarium/wiki/FAQ#reporting-a-crash

**Reply by eriktim82** [reply]
cpu: r9 7950x
rtx 3070
64 gb ram
2tb samsung evo plus nvme ssd

today can only get to run in admin mode

[log.txt](https://github.com/Stellarium/stellarium/files/14807906/log.txt)
